less/grid.less

```diff
@@ -33,7 +33,8 @@
     }
 
     .grid {
-        font-size: 0; // (1)
+        display: flex;
+        flex-wrap: wrap;
         line-height: 1;
```

Comment:
I don't think we want this one, it is necessary to set the line height back then.